Resources/PO/SimpleForm/SimpleFormDemo.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Library    SeleniumLibrary
+Resource    ../../../Data/InputData.robot
```

Comment:
This Resource does not have to be imported here. Better to have the ${INPUT_FORMS_DROPDOWN} variable in this file.